styles/email/_email-base.scss

```diff
@@ -0,0 +1,134 @@
+
+$font-size-unit-convert: false; // Prevents font-related measurements from being converted to the incompatible rem unit
+
+//
+//  Resets
+//  ---------------------------------------------
+
+//  Client-specific resets (from the Salted email template: https://github.com/rodriguezcommaj/salted)
+
+
+//  Prevent WebKit and Windows mobile from changing default text sizes
+body,
+table,
+td,
+a {
+    -ms-text-size-adjust: 100%; // sass-lint:disable-line no-vendor-prefixes
+    -webkit-text-size-adjust: 100%; // sass-lint:disable-line no-vendor-prefixes
+}
+
+//  Allow smoother rendering of resized images in Internet Explorer (such as high-resolution logo)
+img {
+    -ms-interpolation-mode: bicubic; // sass-lint:disable-line no-misspelled-properties no-vendor-prefixes
+}
+
+//  Remove spacing between tables in Outlook 2007 and up
+// sass-lint:disable-all
+table,
+td {
+    mso-table-lspace: 0pt;
+    mso-table-rspace: 0pt;
+}
+// sass-lint:enable-all
+
+//
+//  Typography
+//  ---------------------------------------------
+
+a:visited {
+    color: $link__visited__color !important;
+    text-decoration: $link__visited__text-decoration !important;
+}
+
+a:hover {
+    color: $link__hover__color !important;
+    text-decoration: $link__hover__text-decoration !important;
+}
+
+a:active {
+    color: $link__active__color !important;
+    text-decoration: $link__active__text-decoration !important;
+}
+
+//  Remove link color on iOS
+.no-link a,
+.no-link-a {
+    color: $text__color !important;
+    cursor: default !important;
+    text-decoration: none !important;
+}
+
+//
+//  Layout
+//  ---------------------------------------------
+//  Layout (Mobile)
+@include max-screen($screen__s) {
+    html,
+    body {
+        //  Change background/foreground to same color
+        background-color: $email-content__background-color;
+        width: 100% !important;
+    }
+
+    .main {
+        max-width: 100% !important;
+        min-width: 240px;
+        width: auto !important;
+    }
+}
+
+//
+//  Buttons
+//  ---------------------------------------------
+
+
+.button {
+    .inner-wrapper {
+        td:hover {
+            background-color: $button-primary__hover__background !important;
+        }
+
+        a:active,
+        td:active {
+            background-color: $button-primary__active__background !important;
+        }
+    }
+
+    a:active,
+    a:hover,
+    a:visited {
+        //  Undo general link hover state
+        border: 1px solid $button-primary__hover__background;
+        color: $button-primary__color !important;
+        text-decoration: none !important;
+    }
+}
+
+@include max-screen($screen__xs) {
+    .button {
+        .inner-wrapper {
+            width: 100% !important;
+
+            td {
+                a {
+                    font-size: $button__font-size + 2;
+                }
+            }
+        }
+    }
+}
+
+
+//
+//  Messages
+//  ---------------------------------------------
```

Comment:
What happened here?

Reply:
here could be added Messaages styles that will go to styles tag (can be media queires) but as for now all are in email-inline.scss I removed this comment